released/packages/coq-vellvm/coq-vellvm.1.0.20240416/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+name: "coq-vellvm"
+maintainer: "stevez@cis.upenn.edu"
+synopsis: "Coq library implementing (executable) semantics for LLVM IR"
+
+homepage: "https://github.com/vellvm/vellvm"
+dev-repo: "git+https://github.com/vellvm/vellvm.git"
+bug-reports: "https://github.com/vellvm/vellvm/issues"
+authors: [
+  "Steve Zdancewic <stevez@cis.upenn.edu>"
+  "Yannick Zakowski <yannick.zakowski@inria.fr>"
+  "Calvin Beck <hobbes@seas.upenn.edu>"
+  "Irene Yoon <euisuny@cis.upenn.edu>"
+  "Gary (Hanxi) Chen <hanxic@seas.upenn.edu>"
+]
+license: "GPL-3.0-or-later"
+
+
+build: [make "-C" "src" "all" "-j%{jobs}%"]
+install: [make "-C" "src" "install"]
+    
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "cppo" 
+  "dune" 
```

Comment:
Can you please add a lower bound for the Dune version you support? It can normally be determine by the Coq language version in the `dune` file, e.g., version 0.6 is `"dune" {>= "3.5"}`

Reply:
Gotcha, I will look into this.

Reply:
@palmskog I'm actually not sure about this. In Vellvm we build the Coq code using `coq-makefile` and use `dune` for the extracted ocaml code. I don't believe we use `dune` in a very sophisticated manner, but the `dune` version I use to build the project is `3.13.0`, should I just set the lower bound to that?

Reply:
@Chobbes if you're unsure, maybe just set to something like `"dune" {>= "2.8"}`. There are lots of weird Dune stuff going on below that.

Reply:
Got it, I'll do that. Thanks for all of the help by the way, I really appreciate it :).